### PR TITLE
Bump to Ansible 2.6 and Molecule 2

### PR DESCRIPTION
**Pull request by sbesson**
As discussed earlier today, this new set of requirements supersedes the ones previously proposed in #10.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -68,8 +68,8 @@
     # requirements files see:
     # https://packaging.python.org/en/latest/requirements.html
     install_requires=[
-        'molecule==1.25.1',
-        'ansible==2.3.2',
-        'docker',
+        'ansible==2.6.3',
+        'docker-compose',
```
Comment (manics) [inline setup.py]: Shouldn't this be `docker`?

Comment (sbesson) [reply setup.py]: I copied this requirement from https://github.com/openmicroscopy/ome-ansible-molecule-dependencies/pull/10/files#diff-2eeaed663bd0d25b7e608891384b7298R74 it pulls `docker` as a dependency

```
Collecting docker<4.0,>=3.4.1 (from docker-compose->ome-ansible-molecule-dependencies==0.3.2.dev1)
```

Comment (manics) [reply setup.py]: OK, can you make this a versioned dependency (looks like the current is `1.22.0`), then this should be good to merge

Comment (sbesson) [reply setup.py]: Pushed. 16e4250 also contains the discussed renaming of this meta-package using the proposed `ome-ansible-molecule` suggestion for now although there might be better/shorted/more concise names clarifying the scope of this meta-package.

Once we have the name settled, the repo will need to be renamed before cutting a `0.4.0` release to PyPI and the new package can be consumed across all Ansible roles.

Comment (manics) [reply setup.py]: Thanks, I'll give it a final test with https://github.com/IDR/deployment/pull/135 before merging. `ome-ansible-molecule` sounds fine to me, not overly long but not too generic.

But ***if*** others don't like that name then how about `ome-ansible-deps`?

**Review by manics — APPROVED** [review]
https://github.com/IDR/deployment/pull/135/commits/dc3bc0290a5bdb02299c905a3350433977b124d8 passed travis https://travis-ci.org/IDR/deployment/builds/441763607

**Comment by sbesson** [comment]
Now released as https://pypi.org/project/ome-ansible-molecule/0.4.0/. I will look into whether PyPI has some mechanism to annotate a project as superseded by another. Otherwise, might create a final 0.3.2 release on https://pypi.org/project/ome-ansible-molecule-dependencies/ to indicate the new meta-package should be used instead.

**Pull request merged**